ext/odbc/php_odbc.c

```diff
@@ -974,26 +974,20 @@ PHP_FUNCTION(odbc_execute)
 	unsigned char otype;
 	SQLSMALLINT ctype;
    	odbc_result *result;
-	int numArgs = ZEND_NUM_ARGS(), i, ne;
+	int i, ne = 0;
```

Comment:
Not sure why I added it. So I'll get rid of it.

Reply:
Oh, it has just come to my mind: since the `!pv_param_ht` check below preceds the initialization of `ne` in the next condition, I had to assign it a default value, because it's used inside the "then" branch.